### PR TITLE
Add job page to scenario mgt template

### PR DESCRIPTION
**Pull request by jrobinAV**
NEw page on the scenario mgt template to visualize and manage jobs.

### CODE REVIEW

**Comment by AlexandreSajus** [comment]
I just tested this. Looks good, very easy to understand. It would be helpful if the input node had a small default dataset to run the scenario. I just added a 3-line csv dataset and it was impressive to be able to run the scenario from the app and see job "Completed" and the result

**Review by AlexandreSajus — CHANGES_REQUESTED** [review]
I just tested this. Looks good, very easy to understand. It would be helpful if the input node had a small default dataset to run the scenario. I just added a 3-line csv dataset, and it was impressive to run the scenario from the app and see the job "Completed" and the result.

File: src/taipy/templates/scenario-management/{{cookiecutter.__root_folder_name}}/pages/root.md [hunk 1]
```diff
@@ -9,8 +9,13 @@
 |>
 |>
 
-<|part|class_name=main|render={selected_scenario}|
+<|part|class_name=main|
```
Comment (FlorianJacta) [inline src/taipy/templates/scenario-management/{{cookiecutter.__root_folder_name}}/pages/root.md:12]: What is the use of class_name=main

**Comment by FlorianJacta** [comment]
Excellent PR! Few minor points on this template for a future PR:
- we could put margin to 0 in the run to make it nicer looking:
`gui.run(title="{{cookiecutter.__application_title}}", margin="0em")`

image

- we can delete the lines concerning the issue on `from ... import *`

- we could also use `on_submission_change` callback

File: src/taipy/templates/scenario-management/{{cookiecutter.__root_folder_name}}/pages/root.md (hunk 1 above)
Comment (jrobinAV) [reply src/taipy/templates/scenario-management/{{cookiecutter.__root_folder_name}}/pages/root.md:12]: IDK. Is'nt it used for applying default styling?

**Comment by jrobinAV** [comment]
> I just tested this. Looks good, very easy to understand. It would be helpful if the input node had a small default dataset to run the scenario. I just added a 3-line csv dataset, and it was impressive to run the scenario from the app and see the job "Completed" and the result.

Actually, we removed the dataset recently.

The purpose of a template is not to demonstrate Taipy features nor to teach users how to do an application. The goal is to speed up the development of an application. If we add a dataset, as a user, I would systematically remove the data set right after scaffolding the application. That would be the very first thing to do, so it's slowing down the development.
Instead of adding a dataset, we have an upload control so the user can directly upload its dataset. We also have placeholders that show where to initialize data.

I don't want to change that.

**Comment by jrobinAV** [comment]
> Excellent PR! Few minor points on this template for a future PR:
> 
> * we could put margin to 0 in the run to make it nicer looking:
>   `gui.run(title="{{cookiecutter.__application_title}}", margin="0em")`
> 
> image
> 
> * we can delete the lines concerning the issue on `from ... import *`
> * we could also use `on_submission_change` callback

@FlorianJacta Can you create an improvement ticket for that?

File: src/taipy/templates/scenario-management/{{cookiecutter.__root_folder_name}}/pages/root.md (hunk 1 above)
Comment (FlorianJacta) [reply src/taipy/templates/scenario-management/{{cookiecutter.__root_folder_name}}/pages/root.md:12]: I don't think so

**Comment by AlexandreSajus** [comment]
> > I just tested this. Looks good, very easy to understand. It would be helpful if the input node had a small default dataset to run the scenario. I just added a 3-line csv dataset, and it was impressive to run the scenario from the app and see the job "Completed" and the result.
> 
> Actually, we removed the dataset recently.
> 
> The purpose of a template is not to demonstrate Taipy features nor to teach users how to do an application. The goal is to speed up the development of an application. If we add a dataset, as a user, I would systematically remove the data set right after scaffolding the application. That would be the very first thing to do, so it's slowing down the development. Instead of adding a dataset, we have an upload control so the user can directly upload its dataset. We also have placeholders that show where to initialize data.
> 
> I don't want to change that.

Then it needs to be clear that a dataset needs to be uploaded before I run an execution.

I imagine a user will create this template, try running an execution, and fail with an error saying "no dataset found"; the user will be frustrated and not touch the template again.

**Pull request merged**